style.css

```diff
@@ -0,0 +1,47 @@
+h1 {
+  font-size: 200px;
+  color: white;
+ 
+  
+}
+
+h2 {
+  color: white;
+  text-align: left;
+ 
+   
+
+}
+body {
+  
+  font-family: "Helvetica Neue", sans-serif;
```

Comment:
These styles are relying on a user's fonts and [less than 8% of Windows users have Helvetica on their system](https://www.cssfontstack.com/Helvetica). This was specifically the example used in the class readings on [fonts in CSS](https://mica-web.github.io/learn/css/fonts#web-safe-font-stacks).

If using web safe fonts, you must include a more robust font stack to provide a similar experience for users who do not have this font available to them. Or, as required by this project, you must guarantee they have the font by using a [web font](https://mica-web.github.io/learn/css/fonts#web-fonts) via Google Fonts.